Development/src/components/ActiveField.js

```diff
@@ -33,21 +33,24 @@ const toggleMasterEnable = (record, resource) => {
             id: record.id,
         })
             .then(({ data }) => {
+                if (!data.hasOwnProperty('$staged')) {
+                    throw new Error('Could not reach connection API');
```

Comment:
connection -> Connection

And there are two possibilities, right? Either the IS-04 Device simply doesn't list any Connection API endpoints in its `controls`, or none of them responded? Does 'reach' cover both of those? 🤔

Reply:
Would 'No connection API found' be a suitable message?

Reply:
Yes - with the correct capitalization. 😉